Add Google OAuth configuration to README and .env.dist (#484)
short-d/short: README.md

1. Replace the value of `FACEBOOK_CLIENT_SECRET` in `backend/.env` file with
   `App Secret`.

> ### Create Google OAuth Application

Reviewer: ```suggestion
### Create Google OAuth Client ID
```

> Create a new credential at

Reviewer: ```suggestion
Create a new client ID at
```

>    [Credentials page](https://console.developers.google.com/apis/credentials) with the following steps:

Reviewer: ```suggestion
   [Google API Credentials](https://console.developers.google.com/apis/credentials):
```

> 1. Click `OAuth client ID` under `Create credentials`.

Reviewer: ```suggestion
1. Click on `Create Credentials` and select `OAuth client ID`.
```

   ![Credentials](doc/google/credentials.jpg)

> 1. Choose `Web application` under `Application type`.

Reviewer: ```suggestion
1. Select `Web application` for `Application type`.
```

> 1. Put `http://localhost/oauth/google/sign-in/callback` under `Authorized redirect URIs`, click `Create`.

Reviewer: ```suggestion
1. Fill in `http://localhost/oauth/google/sign-in/callback` for `Authorized redirect URIs` and click on `Create`
```

   ![OAuth](doc/google/oauth.jpg)
   

> 1. Replace the value of `GOOGLE_CLIENT_ID` in `backend/.env` file with `Client ID`.

Reviewer: ```suggestion
1. Replace the value of `GOOGLE_CLIENT_ID` in `backend/.env` file with `Your Client ID`.
```

Author: addressed comment

1. Replace the value of `GOOGLE_CLIENT_SECRET` in `backend/.env` file with

>    `Client Secret`.

Reviewer: ```suggestion
   `Your Client Secret`.
```
